### PR TITLE
Defi Kingdoms - new potion addresses

### PR DESCRIPTION
**Pull request by antonyip**
Left a simple script to extract token_name, token_symbol and token_decimals

### CODE REVIEW

File: data/defikingdoms_dex/tokens.py [hunk 1]
```diff
@@ -19,3 +19,17 @@
...
+    print('0x1771dec8d9a29f30d82443de0a69e7b6824e2f53,Anti-Blind Potion,DFKANTBLND,0')
+
+if not ('0x7e120334d9affc0982719a4eacc045f78bf41c68' in dups):
+    print('0x7e120334d9affc0982719a4eacc045f78bf41c68,Magic Resistance Potion,DFKMGCRSPTN,0')
+
+if not ('0xfb03c364969a0bb572ce62b8cd616a7ddeb4c09a' in dups):
+    print('0xfb03c364969a0bb572ce62b8cd616a7ddeb4c09a,Toughness Potion,DFKTFNSPTN,0')
+
+if not ('0x872dd1595544ce22ad1e0174449c7ece6f0bb01b' in dups):
+    print('0x872dd1595544ce22ad1e0174449c7ece6f0bb01b,Swiftness Potion,DFKSWFTPTN,0')
```
Comment (chuxinh) [inline data/defikingdoms_dex/tokens.py]: add one line
wow so you have to add manually like these if there's new tokens?

File: data/harmony_tokens/run.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#     "params": [
+#         {
+#             "to": "0xA1f8b0E88c51a45E152934686270DDF4E3356278",
+#             "data": "0x313ce567"
+#         },
+#         "latest"
+#     ],
+#     "id": 1
+# }'
+#
```
Comment (chuxinh) [inline data/harmony_tokens/run.sh]: remove this line

**Review by chuxinh — APPROVED** [review]
LGTM after nits

File: data/defikingdoms_dex/tokens.py (hunk 1 above)
Comment (antonyip) [reply data/defikingdoms_dex/tokens.py]: For now.. until I find a better way.. https://github.com/MetricsDAO/harmony_dbt/issues/38

**Pull request merged**